.github/workflows/tests.yaml

```diff
@@ -20,6 +20,9 @@ jobs:
         python-version: ["3.9", "3.10", "3.11"]
 
     steps:
+      - name: Support long paths
```

Comment:
This is to avoid `File name too long` error when checking out the repo in CI. Generated data from DAT project has long file names, plus hierarchy levels.